content/systems/filecoin_vm/runtime/_index.md

```diff
@@ -12,26 +12,36 @@ dashboardTests: 0
 
 ## Receipts
 
-A `MessageReceipt` contains the result of a top-level message execution.
+A `MessageReceipt` contains the result of a top-level message execution. Every syntactically valid and correctly signed message can be included in a block and will produce a receipt from execution. 
 
 A syntactically valid receipt has:
 
 - a non-negative `ExitCode`,
-- a non empty `ReturnValue` only if the exit code is zero,
+- a non empty `Return` value only if the exit code is zero, and
 - a non-negative `GasUsed`.
 
-## `vm/runtime` interface
+```go
+type MessageReceipt struct {
+	ExitCode exitcode.ExitCode
+	Return   []byte
+	GasUsed  int64
+}
+```
 
-{{<embed src="/externals/specs-actors/actors/runtime/runtime.go" lang="go" >}}
+## `vm/runtime` Actors Interface
 
-## `vm/runtime` implementation
+{{<embed src="/externals/specs-actors/actors/runtime/runtime.go" lang="go" >}}
 
-{{<embed src="impl/runtime.go" lang="go" >}}
+## `vm/runtime` VM Implementation
 
-## Code Loading
+{{<embed src="/externals/lotus/chain/vm/runtime.go" lang="go" >}}
```

Comment:
Here is probably better to include just a github link to that file or maybe just the `Runtime` symbols plus a github link.

Lets try to avoid embed full file now that we have symbols lookup ok ?